tests/commands/test_runner.py

```diff
@@ -86,16 +86,16 @@ def test_integration(self):
 
     def test_debug_enabling(self):
         """
-        {DD,DATADOG}_TRACE_DEBUG=true allows setting debug logging of the global tracer
+        DD_TRACE_DEBUG=true allows setting debug logging of the global tracer.
         """
-        with self.override_env(dict(DATADOG_TRACE_DEBUG="false")):
```

Comment:
Probably worth keeping some tests with the old variable around to ensure we don't break backwards compatibility